Voilà ta branche où tu peux faire ce que tu veux (test fork ?)

amostin opened the pull request:
### Faut qu'on définisse quand on va la fermer parce que on garde en tête d'ajouter des fonctionalités petit a petit qui s'intègre facilement 😄 
> 👉 Donc quand tu fini ce sur quoi tu travailles actuellement:
> * tu merge la branche (ou tu demande si on est d'accord pour merger)
> * puis tu peux créer une nouvelle branche pour travailler sur une nouvelles fonctionnalité du programme 

> 👉 Typiquement, si tu veux taffer sur le fork(), c'est mieux de créer une nouvelle branche comme ça on garde la branche version_2 à jour avec ton code et nous si on travaille sur d'autre branche on reçoit tes mises à jour

amostin committed:
voilà le fichier où tu travailles est à sa place

ATHOOS commented:
Ok donc, mnt je peux encore bosser sur cette branche ci, genre peaufiner les fonctions, etc, et une fois que c'est fait, je vous le soumets pour qu'on merge ? Et si je veux passer a de nouvelles fonctionalites, je crée une nouvelle branche a partir de celle-ci afin de reprendre de la on ou en etait tout en gardant une trace de ce qu'on a fait ?

amostin commented:
@ATHOOS 
Tas bien résumé mais le mieux cest de merge cette branche avec version_2 puis creer une nouvelle branche pour travailler sur une autre fonction.

## Le truc c'est d'essayer de travailler sur une chose a la fois :D comme ça si je veux prendre un bout de ton code mais pas tout, je peux le reprendre via ta branche.

> Donc tu peux faire une branche sur ta branche mais je pense que plus on seloigne de master plus ça craint donc essayons de rester proche de version_2 qui elle meme est proche de master :D

### c'est comme si on devait faire passer notre code par un sas (version_2) qui verifie que la fonctionalité seul est bien puis par un autre sas (master) qui verifie que lensemble des fonctionnalité sont bien :D

amostin commented:
Désolé jarrive pas a être concis c'est pas totalement clair pour moi non plus :D

amostin commented:
Je me permet de décider mais si tu trouves quon devrait structurer autrement n'hésite pas je suis ***totalement*** ouvert aux changements !

ATHOOS commented:
Pour moi c'est niquel, juste qu'on s'accorde bien sur les merge, etc.

ATHOOS commented:
Mais donc la tu veux que je merge avec version_2 c'est bien ça ?

ATHOOS committed:
code affiné, décalage d'affichage corrigé, bug de format corrigé

code allégé, les cas de décalage lors du printf sont corrigés, et correction de la fonction affchage() afin d'afficher tout les temps traduits correctement

amostin commented:
## Ch'ui chaud merge
### Si je comprend bien: la voiture tourne, affiche les temps bien formaté (en tenant compte de la longeur de la chaine a afficher pour que le tableau reste droit) jusqu'à la *fin de la course* ou *sortie de route*, tout en tenant en compte le temps tour total, le numero du tour, pit et out ? 
> **Dis moi si j'ai loupé une fonctionnalité que tas ajouté**
